Validation/RecoVertex/python/PrimaryVertexAnalyzer4PUSlimmed_cfi.py

```diff
@@ -27,7 +27,13 @@
                                                                       "selectedOfflinePrimaryVertices",
                                                                       "selectedOfflinePrimaryVerticesWithBS"
                                                                       ),
+                               nPUbins = cms.uint32(130)
```

Comment:
this should be `int32` as per declaration https://github.com/cms-sw/cmssw/pull/39237/files#diff-e6e5e0e7d53a3ec3ac6b190418a85213e8dee46a1dfb647c44a8d80d8f5d93dbR184 (or viceversa)